starter-code/css/main.css

```diff
@@ -6,3 +7,164 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+  font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;*/
+}
+
+.navbar{
+  top:0px;
+  width:98.6%;
+  background:White;
+  display:flex;
+  justify-content: flex-start;
+  position:fixed;
+  border-bottom: 1px solid black;
+  z-index:1;
+}
+
+.iconspotify{
+  display:block;
+  padding-top: 10px;
+  padding-bottom: 20px;
+  padding-left: 30px;
+}
+
+.iconspotify:hover {
+ animation-name : bounce;
+ position: relative;
+ animation-iteration-count : infinite;
+ animation-duration: 2s;
+}
+
+@keyframes bounce {
+  0% {
+    margin-top : 50px;
+  }
+  50% {
+    margin-top : -20px;
+  }
+  100%{
+    margin-top : 50px;
+  }
+}
+
+
+ul {
+  list-style:none;
+  position: absolute;
+  text-indent: 2em;
+  margin-left: 625px;
+  padding-top: 35px;
+}
+
+ul li {
+  display: table-cell;
+  color: black;
+  opacity: 0.8;
+  font-size: 18px;
+}
+
+.backgroundbloc{
+  height: 720px;
+  background-image: url(../images/landing.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position-y: center;
+  background-position-x: center;
+}
+
+.baselinegroup {
+  position: relative;
+  color: white;
+  padding-left: 50px;
+  margin-bottom: 300px;
+  top:250px; width:1000px; height:20px;
+  text-align: center;
+}
+
+h1{
+  font-size: 65px;
+  padding-top:10px;
+}
+
+.paragraph {
+  font-size: 25px;
+  margin-top: -40px;
```

Comment:
Negative margins are a red flag. If you **need** them, sure, go ahead and use them. But I'm sure there is simpler solution.